src/org/opendatakit/briefcase/ui/export/ExportPanelForm.form

```diff
@@ -51,15 +51,6 @@
             <properties/>
             <border type="none"/>
             <children>
-              <component id="b7dc5" class="javax.swing.JLabel" binding="exportingLabel">
```

Comment:
As we talked over on Slack, the layout needs some work, and remember to rebuild after making changes on the UI Designer ;)